### DISCUSSION TITLE
Dúvida do Curso de Lógica de Programação com DART

### DISCUSSION THREAD

**Discussion by MateuzAndrade** [balta-io/forum#328] [Tira Dúvidas] [answered]
Hey Pessoal , Tudo Bem?

Alguém sabe me dizer a extensão do VS CODE que o GABU usa no curso de Lógica de Programação em DART, esta extensão adiciona comentários sobre a construção dos itens, exemplo, ESTÁ FALTANDO UM ; NO FINAL DA DECLARAÇÃO.

Abraços

**Comment by andrebaltieri** [comment, marked as answer]
@MateuzAndrade acredito que seja a Error Lens!

**Reply by MateuzAndrade** [reply]
Obrigado Balta

**Reply by andrebaltieri** [reply]
🩷🚀
